.travis.yml

```diff
@@ -10,35 +10,20 @@ cache:
 
 language: ruby
 rvm:
-  - 1.8.7
-  - 1.9.3
```

Comment:
We can skip all rails tests if rails are not in Gemfile, like

```ruby
begin
  require "rails/all"
rescue
  puts "skipping rails tests"
  exit
end
```

Plus crate `gemfiles/no_rails.gemfile` and target non-rails ruby versions. Though I want to notice, that ruby 1.9.3 is not supported officially and its use discouraged.